content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+   ```
+
+#### Merge conflicts and rebasing
+
+If another contributor commits changes to the same file in another PR, it can
+create a merge conflict. You must resolve all merge conflicts in your PR.
+
+1. Update your fork and rebase your local branch:
+
+   ```shell
+   git fetch origin
+   git rebase origin/<your-branch-name>
+   ```
+
+   Then force-push the changes to your fork:
+
+   ```shell
+   git push --force-with-lease origin <your-branch-name>
+   ```
+
+1. Fetch changes from `open-telemetry/opentelemetry.io`'s `upstream/main` and
+   rebase your branch:
+
+   ```shell
+   git fetch upstream
+   git rebase upstream/main
+   ```
+
+1. Inspect the results of the rebase:
+
+   ```shell
+   git status
+   ```
+
+   This results in a number of files marked as conflicted.
+
+1. Open each conflicted file and look for the conflict markers: `>>>`, `<<<`,
+   and `===`. Resolve the conflict and delete the conflict marker.
+
+   For more information, see
+   [How conflicts are presented](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented).
+
+1. Add the files to the changeset:
+
+   ```shell
+   git add <filename>
+   ```
+
+1. Continue the rebase:
+
+   ```shell
+   git rebase --continue
+   ```
+
+1. Repeat steps 2 to 5 as needed.
+
+   After applying all commits, the `git status` command shows that the rebase is
+   complete.
+
+1. Force-push the branch to your fork:
+
+   ```shell
+   git push --force-with-lease origin <your-branch-name>
+   ```
+
+   The pull request no longer shows any conflicts.
+
+## Contribute to other repositories
+
+The OpenTelemetry project contains many repositories. Many of these repositories
+contain documentation: user-facing help text, error messages, API references or
+code comments.
+
+If you see text you'd like to improve, use GitHub to search all repositories in
+the OpenTelemetry organization. This can help you figure out where to submit
+your issue or PR.
+
+Each repository has its own processes and procedures. Before you file an issue
+or submit a PR, read that repository's `README.md`, `CONTRIBUTING.md`, and
+`code-of-conduct.md`, if they exist.
+
+Most repositories use issue and PR templates. Have a look through some open
+issues and PRs to get a feel for that team's processes. Make sure to fill out
+the templates with as much detail as possible when you file issues or PRs.
+
+## Open an issue
+
+If you want to suggest improvements to existing content or notice an error, open
+an issue.
+
+1. Click the **Create documentation issue** link on the right sidebar. This
+   redirects you to a GitHub issue page prepopulated with some headers.
+2. Describe the issue or suggestion for improvement. Provide as many details as
+   you can.
+3. Click **Submit new issue**.
+
+After submitting, check in on your issue occasionally or turn on GitHub
+notifications. Reviewers and other community members might ask questions before
+they can take action on your issue.
```

Comment:
Add a few words about patience, that it can take a few days until maintainers respond. If they don't get an answer within 7 days they can bump the issue up with a message or reach out via slack